### PR TITLE
Add task to create csv file for syncronization tasks

### PR DESCRIPTION
**Pull request by Bogdan-Wiederspan**
This PR will add a task called CreateSyncFile, which creates for given columns and mapping scheme a csv file necessary to use synchronization tools.

This PR comes in a 2-step process:
Part 1: Add task structure to generally create the csv file from existing columns
Part 2: add structure to handle selection masks and also save them within a csv

### CODE REVIEW

**Review by riga — CHANGES_REQUESTED** [review]
Some minor changes.
Let's discuss details on the task implementation in person.

File: hbt/tasks/sync_csv.py [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+        ).rename(columns=mapping, inplace=False)
+        self.output().dump(merged_pandas_framework, index=False, formatter="pandas")
+
+
+# overwrite class defaults
+check_finite_tasks = law.config.get_expanded("analysis", "check_finite_output", [], split_csv=True)
+CreateSyncFile.check_finite_output = ChunkedIOMixin.check_finite_output.copy(
+    default=CreateSyncFile.task_family in check_finite_tasks,
+    add_default_to_description=True,
+)
```
[the remark this answers is not in the document]
Comment (Bogdan-Wiederspan) [reply hbt/tasks/sync_csv.py:251]: what about check_overlap_task right below?

File: hbt/tasks/sync_csv.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
+# coding: utf-8
+
+"""
+Task to create a single file csv file for framework sync with other frameworks.
+"""
+
+import luigi
+import law
+
```
[the remark this answers is not in the document]
Comment (Bogdan-Wiederspan) [reply hbt/tasks/sync_csv.py:9]: done, im starting to think that i use flake8 wrong^^

File: hbt/tasks/sync_csv.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (riga) [reply hbt/tasks/sync_csv.py]: This check is input related and should / can run.

[the remark this answers is not in the document]
Comment (Bogdan-Wiederspan) [reply hbt/tasks/sync_csv.py]: ok, is done.

**Review by riga — APPROVED** [review]

**Pull request merged**